kodilla-stream/src/test/java/com/kodilla/stream/portfolio/BoardTestSuite.java

```diff
@@ -135,4 +141,21 @@ public void testAddTaskListFindLongTasks() {
         //Then
         Assert.assertEquals(2, longTasks);
     }
+
+    @Test
+    public void testAddTaskListAverageWorkingOnTask() {
+        //Given
+        Board project = prepareTestData();
+
+        //When
+        double averageWorkingOnTask = project.getTaskLists().stream()
+                .filter(line -> "In progress".equals(line.getName()))
```

Comment:
Zastanawiałeś się, czemu lepiej zastosować `"In progress".equals(foo)` niż `foo.equals("In progress")`, mimo iż to drugie jest (przynajmniej subiektywnie dla mnie) czytelniejsze?